### PR TITLE
User can see remaining GitHub API calls for the hour #771

### LINKED ISSUE #771: User can see remaining GitHub API calls for the hour
Related to #765.

Will be an additional option on the Preferences tab. Tentatively, change "Preferences" to something else - currently it just has Logout and Quit.

### PR DESCRIPTION
**Pull request by acruis**
Fixes #771

### CODE REVIEW

File: src/main/java/ui/MenuControl.java [hunk 1]
```diff
@@ -280,8 +284,30 @@ private MenuItem createRefreshMenuItem() {
     public void scrollTo(int panelIndex, int numOfPanels){
         setHvalue(panelIndex * (panelsScrollPane.getHmax()) / (numOfPanels - 1));
     }
+
     private void setHvalue(double val) {
         panelsScrollPane.setHvalue(val);
     }
 
+    private void showDialogOnAPICheck(int remaining, LocalDateTime reset, Throwable ex) {
```
Comment (lejolly) [inline src/main/java/ui/MenuControl.java]: Please remove if it's not in use.

File: src/main/java/util/DialogMessage.java [hunk 2]
```diff
@@ -30,4 +30,11 @@ public static void showErrorDialog(String header, String message){
         return result.get().equals(yesButton);
     }
 
+    public static void showInfoDialog(String header, String message) {
```
Comment (lejolly) [inline src/main/java/util/DialogMessage.java]: Is this used?

Comment (acruis) [inline src/main/java/util/DialogMessage.java]: Oh, forgot about that.

**Comment by lejolly** [comment]
Looks good.

**Pull request merged**